common/src/main/java/io/netty5/util/HashedWheelTimer.java

```diff
@@ -115,6 +115,7 @@ public class HashedWheelTimer implements Timer {
     private final AtomicLong pendingTimeouts = new AtomicLong(0);
     private final long maxPendingTimeouts;
     private final Executor taskExecutor;
+    private long currRound;
```

Comment:
I am wondering, shouldn't this field be moved to `Worker` class where it belongs? (otherwise it is potentially shared between multiple threads and may need `volatile`, seems to be unnecessary)

Reply:
There is a logic error in my code. Please refer to #13018 for detail.